src/content/en/updates/2019/01/shape-detection.md

```diff
@@ -2,20 +2,23 @@ project_path: /web/_project.yaml
 book_path: /web/updates/_book.yaml
 description: The Shape Detection API allows for faces, barcodes, and text to be detected in images.
 
-{# wf_updated_on: 2019-02-22 #}
+{# wf_updated_on: 2019-08-06 #}
 {# wf_published_on: 2019-01-07 #}
 {# wf_tags: capabilities,shape-detection,progressive-web-apps,webapp #}
 {# wf_featured_image: /web/updates/images/generic/timeline.png #}
 {# wf_featured_snippet: The Shape Detection API allows for faces, barcodes, and text to be detected in images. #}
-{# wf_blink_components: Blink>ImageCapture #}
+{# wf_blink_components: Blink>ShapeDetection #}
 
 # A Picture is Worth a Thousand Words, Faces, and Barcodes—The Shape Detection API {: .page-title}
 
 {% include "web/_shared/contributors/thomassteiner.html" %}
 
 <div class="clearfix"></div>
 
-Warning: We’re currently working the specification for this API as part of the [capabilities
+Success: Following a successful Origin Trial,
```

Comment:
'Origin trial' is lower case. Please fix this throughout.

Reply:
Done.